Add support for `<optgroup>` to the `Select` component (#395) (#401)
react-ui-org/react-ui: src/lib/components/SelectField/README.mdx

  }}
</Playground>

## Grouping Related Options

> In case of a large amount of options, consider **grouping related items
> together** by nesting them (implements the `optgroup` HTML element).

Reviewer: Now when I see it live and in context, I'd change the texts after all. I believe this will work better:

1. Please keep the original hint in the _General Guidelines_ section, just shorter. The advice still has its place there.

   ```
   In case of a large amount of options, consider **grouping related items
   together** by nesting them.
   ```

2. Please change the text here so it works more like API description:

   ```suggestion
   For a large amount of options you can group related items together
   by nesting them (implements the `optgroup` HTML element).
   ```
